res/sections/42-ComandiBase.tex

```diff
@@ -0,0 +1,109 @@
+l'ambiente sarà \verb!\begin{wrapfig}{pos}{dim}...\end{wrapfig}! dove \texttt{
+pos} può essere \textbf{r} o \textbf{l} e \texttt{dim} serve per specificare 
+le dimensioni di quanto l'immagine deve ``affiancarsi'' al testo. Di seguito un 
+esempio. 
+\lstinputlisting{res/examples/esempiotestoafianco.tex}
+
+
+\section{Note a piè di pagina}
+Spesso capita di dover mettere delle note a piè di pagina, per esempio per mettere un riferimento o spiegare meglio qualcosa\footnote{O solo per distrarre il lettore, come in questo caso}. Per fare ciò è necessario utilizzare il comando \verb!\footnote{...}! specificando tra graffe il testo della nota. \LaTeX{} in automatico si occuperà della numerazione e del posizionamento.
+
+
+\section{Collegamenti ipertestuali}
+
+\subsection{Collegamenti nel testo}
+Quando si inseriscono delle immagini e se ne fa riferimento nel testo oppure quando si scrive una parte e la si riprende in un'altra sezione del documento si sente la necessità di inserire un collegamento tra le due parti. Per fare ciò è necessario marcare con \verb!\label{...}! a cui si vuole fare riferimento. Tra parentesi graffe un identificativo univoco all'interno del documento. Quando ci si riferisce ad esso, si deve utilizzare \verb!\ref{...}!, specificando tra graffe lo stesso identificativo. È possibile anche fare riferimento alla pagina in cui si trova l'oggetto marcato con \verb!label! utilizzando \verb!\pageref{...}! (e lo stesso identificativo).
+
+\subsubsection{Buone regole per gli identificativi}
+Solitamente, agli identificativi utilizzati dalle \verb!label! e \verb!ref! vengono anteposti alcuni caratteri che permettono di identificare che tipo di oggetto è referenziato. 
+\begin{table}[]
+\centering
+\begin{tabular}{ll}
+\hline
+\textbf{prefisso} & \textbf{tipologia}         \\ \hline
+sec:              & section                    \\ \hline
+subsec:           & subsection                 \\ \hline
+fig:              & figure                     \\ \hline
+tab:              & tabella                    \\ \hline
+eq:               & formula                    \\ \hline
+lst:              & codice                     \\ \hline
+itm:              & elementi di lista numerata \\ \hline
+alg:              & algoritmo                  \\ \hline
+app:              & appendice                  \\ \hline
+\end{tabular}
+\end{table}
+
+\subsection{Collegamenti al Web}
+Il pacchetto che permette l'inserimento in maniera corretta in un documento \LaTeX{} di \textit{link} o \textit{URL} è \verb!hyperref!. Ci sono due modalità principali per inserire queste informazioni. La prima è \verb!\url{indirizzo}!, con la quale verrà mostrato l'URL in formato \textit{monospace}, pensato anche per documenti che devono essere stampati, la seconda è \verb!\href{indirizzo}{testo_da_mostrare}!, la quale non mostrerà esplicitamente l'indirizzo, ma mostrerà solamente il testo voluto (come succede nei siti Web per capirci). 
```

Comment:
- 80 colonne
- Qui aggiungerei che quindi se si ha intenzione stampare o distribuire in forma cartacea il PDF è meglio quindi usare `url` al posto di `href` se non si vuole far perdere al lettore informazione